hieradata_aws/class/staging/db_admin.yaml

```diff
@@ -43,6 +43,17 @@ govuk_env_sync::tasks:
     temppath: "/tmp/local-links-manager_production"
     url: "govuk-production-database-backups"
     path: "postgresql-backend"
+  "push_local-links-manager_staging_daily":
+    ensure: "present"
+    hour: "4"
+    minute: "30"
+    action: "pull"
```

Comment:
There's an inconsistency here between the action and the name above "push_local-links-manager_staging_daily"

Reply:
will be fix here: https://github.com/alphagov/govuk-puppet/pull/10342 thanks!